packages/tools/client-debugger/client-debugger/src/test/ClientDebugger.test.ts

```diff
@@ -119,6 +119,28 @@ describe("ClientDebugger unit tests", () => {
 		expect(audienceRemoved).to.be.true;
 	});
 
+	it("Validate container state contents are as expected ", async () => {
```

Comment:
Looks great! Might be good to add a `dispose` call to the end and verify we logged a disposal event, though.

Reply:
Added.